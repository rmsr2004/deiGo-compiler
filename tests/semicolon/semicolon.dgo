<<<<<<< HEAD
compiladores 
e 
fixe
=======
compiladores e fixe
>>>>>>> e785b6e890d93922ab27064f000bbd998417161d
return
)
]
}
<<<<<<< HEAD
=======
"compiladores podia ser melhor"
/*
"vou fazer compiladores este ano"
*/
// sonhar nao custa
1.e+0 
"o melhor numero e" : 42
>>>>>>> e785b6e890d93922ab27064f000bbd998417161d
