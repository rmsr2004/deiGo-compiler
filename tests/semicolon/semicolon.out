IDENTIFIER(compiladores)
IDENTIFIER(e)
IDENTIFIER(fixe)
SEMICOLON
RETURN
SEMICOLON
RPAR
SEMICOLON
RSQ
SEMICOLON
RBRACE
SEMICOLON
STRLIT("compiladores podia ser melhor")
SEMICOLON
DECIMAL(1.e+0)
SEMICOLON
STRLIT("o melhor numero e")
Line 12, column 21: illegal character (:)
NATURAL(42)
<<<<<<< HEAD
SEMICOLON
=======
SEMICOLON
>>>>>>> e785b6e890d93922ab27064f000bbd998417161d
